src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,59 @@
+import { NavLink } from "react-router-dom";
+import "./Navbar.css";
+import BorderAllOutlinedIcon from "@material-ui/icons/BorderAllOutlined";
+import LocalMallOutlinedIcon from "@material-ui/icons/LocalMallOutlined";
+import FavoriteBorderOutlinedIcon from "@material-ui/icons/FavoriteBorderOutlined";
+import PersonOutlineOutlinedIcon from "@material-ui/icons/PersonOutlineOutlined";
+
+export default function Navbar() {
+  return (
+    <>
+      <nav className="navbar">
+        <NavLink
+          end
+          to="/"
+          activeClassName="selectedNavItem"
+          className="navItem"
+        >
+          <BorderAllOutlinedIcon
+            classes={{ root: "navIcons" }}
+            style={{
+              transform: "rotateZ(-15deg)",
+            }}
+          />
+          Dice
+        </NavLink>
+
+        <NavLink
```

Comment:
It'd could look even better if you use the _icons_ for respective items on navbar for larger screens too.

Reply:
I tried, But it was not looking good.